### ISSUE TITLE
[feature request] systemd user timer

### ISSUE THREAD

**Issue by ecoutu** [roddhjav/pass-tomb#19]
Briefly reading through the code to pass-tomb, I see you use `sudo` to create systemd system timers. Is there any reason you did not consider using `--user` mode timers?

I believe it would also allow removing the requirement to set permissions, which uses `sudo`, which would entirely remove it as a dependency.

**Comment by roddhjav** [comment]
Hi, 

This is not possible. The timer must be a system timer because `pass close` requires `sudo`. This is a requirement from [Tomb](https://github.com/roddhjav/Tomb/blob/master/tomb#L2651). In the same way, `pass tomb` (via Tomb) also requires `sudo` to generate the Tomb therefore it will always remain a dependency.

**Comment by ecoutu** [comment]
I did not need to use `sudo` to run `pass tomb`, nor did I need it for `pass close`. If you look at my pull request, it was working on Ubuntu 18.04 (I'm running a combination of my two branches I opened P/Rs for on my local machine without issue). 

Maybe I'm missing something, it could be an environment / version thing. 

If you do not want to merge some form of the pull request referenced (I'm happy to add it as an argument to pass-tomb to use `--user` instead of `--system`) I'll continue to use my fork. 

Thanks for your time and effort on this project. I'm using zfs with send / receive backups to rsync.net, which requires running the remote backup system in a jail / VM / not quite sure. Anyways, they have full access to your data from what I can tell. I'm so far manually encrypting the important things with gpg, but obfuscating the store / directory structure itself is something I've actually thought about - I have many accounts - the directory structure of passwords / 2fa backup codes for all of them would paint a pretty picture of who I am, on the entire web. Note: I totally trust rsync.net, I'm pretty sure they have many standards compliance in practice, but my lizard brain still mistrusts a bad actor.
